src/Avalonia.Controls/SplitButton/SplitButton.cs

```diff
@@ -0,0 +1,709 @@
+﻿using System;
+using System.Reactive.Disposables;
+using System.Windows.Input;
+using Avalonia.Controls.Metadata;
+using Avalonia.Controls.Primitives;
+using Avalonia.Input;
+using Avalonia.Interactivity;
+using Avalonia.LogicalTree;
+
+namespace Avalonia.Controls
+{
+    /// <summary>
+    /// A button with primary and secondary parts that can each be pressed separately.
+    /// The primary part behaves like a <see cref="Button"/> and the secondary part opens a flyout.
+    /// </summary>
+    [PseudoClasses(
+        ":disabled",
+        ":secondaryButtonRight",
```

Comment:
Should these be camel case or all lowercase? I'm guessing based on ":pointerover" existing in code now that all should be lowercase.

Reply:
Lowercase. See https://github.com/AvaloniaUI/Avalonia/blob/0af5cd692aeef13d56a194ee55a6c40634d9752c/src/Avalonia.Input/InputElement.cs#L19

Reply:
Ah, thanks, I was looking for better examples. So I should be adding a hyphen "-" between words then same as CSS. Makes some sense. Its just different than XAML itself which follows PascalCase the same as C# (UWP VisualStateManager).

Reply:
Should be resolved: All names converted to lowercase with dashes between words (pointerover kept as one word)

Reply:
All names completely changed and reworked with latest commits. Doesn't apply much anymore.